### PR TITLE
Moved changes for PFM 1723

### PR DESCRIPTION
**Pull request by hariomsinha-eGov**

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/css/src/components/card.scss [hunk 1]
```diff
@@ -0,0 +1,401 @@
+.card {
+    @apply bg-white m-sm px-md pt-md pb-lg shadow-card;
+    border-radius: 4px;
```
Comment (nipunarora-eGov) [inline frontend/micro-ui/web/micro-ui-internals/packages/css/src/components/card.scss:3]: @hariomsinha-eGov increment css version

File: frontend/micro-ui/web/micro-ui-internals/packages/css/src/components/radiobtn.scss [hunk 2]
```diff
@@ -38,6 +38,15 @@
     @apply block bg-primary-main h-3 w-3 rounded-full absolute top-6 left-6;
   }
 }
+
+
+@screen sm {
+  .radio-wrap {
+    @apply flex-col;
+    gap : 0px !important;
```
Comment (prachirane-eGov) [inline frontend/micro-ui/web/micro-ui-internals/packages/css/src/components/radiobtn.scss:46]: if possible, can we avoid using '!important' ?

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/Masters/src/components/MastersCard.js [hunk 3]
```diff
@@ -14,7 +14,12 @@ const MastersCard = () => {
         label: t("ACTION_TEST_MASTERS"),
         link: `/${window?.contextPath}/employee/masters/search-organization`,
         roles: [],
-      }
+      },
+      {
+        label: t("MASTERS_REGISTER_WAGESEEKER"),
```
[the remark this answers is not in the document]
Comment (hariomsinha-eGov) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/Masters/src/components/MastersCard.js:19]: Thanks, Done.

File: frontend/micro-ui/web/micro-ui-internals/packages/css/src/components/radiobtn.scss (hunk 2 above)
Comment (hariomsinha-eGov) [reply frontend/micro-ui/web/micro-ui-internals/packages/css/src/components/radiobtn.scss:46]: There are few inline css on that element. We need important in this case to override them.

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/Masters/src/configs/configWageSeekerRegistrationForm.js [hunk 4]
```diff
@@ -0,0 +1,475 @@
...
+                masterName: "SocialCategory",
+                moduleName: "common-masters",
+                localePrefix: "MASTERS",
+              },
+            },
+          },
+          {   
+            label: ("PHOTOGRAPH"),
+            populators: (
+              <UploadFile
```
[the remark this answers is not in the document]
Comment (hariomsinha-eGov) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/Masters/src/configs/configWageSeekerRegistrationForm.js]: I have done the respective changes.

[the remark this answers is not in the document]
Comment (hariomsinha-eGov) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/Masters/src/configs/configWageSeekerRegistrationForm.js]: Used Multiupload component and added customstyles props.

**Review by prachirane-eGov — APPROVED** [review]

**Review by jagankumar-egov — APPROVED** [review]

**Pull request merged**